api/src/main/java/org/openmrs/module/idgen/service/IdentifierSourceService.java

```diff
@@ -265,6 +265,14 @@
      */
     @Transactional(readOnly=true)
     IdentifierSource getIdentifierSourceByUuid(String uuid);
+    
+    /**
+     * Queries the database for the identifier source(s) with identifier type: patientIdentifierType 
+     * @param patientIdentifierType

```

Comment:
You also need to document the above parameter.

Reply:
documented